.github/workflows/ci-main.yml

```diff
@@ -225,58 +233,11 @@ jobs:
       - name: prepare variables
         uses: ./.github/actions/docker-push-variables
         id: ev
-        env:
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
@BeryJu I removed this to avoid passing the secret around for no reason. Instead, I added a check for `github.repository == 'goauthentik/authentik'`, but the documentation isn't clear on whether this will work for forks. Do you have any insight on this?